### PR TITLE
Add saving out BatchML batch

### LINKED ISSUE #59: Save one batch to s3
## Detailed Description
Would be good to save one batch to s3. This means we could analysis and do checks that the data going into the forecast model is correct

## Possible Implementation
Have option in make_batches to save first batch to a folder (which could be s3). Could also save all batches, if the size is not to big

### PR DESCRIPTION
**Pull request by jacobbieker**
# Pull Request

## Description

Adds saving out the BatchML dictionary to a location for the first batch, so that it can be used for debugging/visualization

Fixes #59 

## How Has This Been Tested?

_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration_

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by peterdudfield** [comment]
do you need to bump the dataloader version?

File: nowcasting_forecast/app.py [hunk 1]
```diff
@@ -124,7 +124,9 @@ def run(
                     )
                 elif model_name == "cnn":
                     dataloader = get_cnn_data_loader(
-                        src_path=f"{temporary_dir}/live", tmp_path=f"{temporary_dir}/live"
+                        src_path=f"{temporary_dir}/live",
+                        tmp_path=f"{temporary_dir}/live",
+                        batch_save_dir=batch_save_dir,
```
Comment (peterdudfield) [inline nowcasting_forecast/app.py:129]: do you need to add this to get_cnn_data_loader too?

**Comment by jacobbieker** [comment]
> do you need to bump the dataloader version?

Yep, done for both now

File: nowcasting_forecast/app.py (hunk 1 above)
Comment (jacobbieker) [reply nowcasting_forecast/app.py:129]: Done! Just had forgotten to push that change

**Pull request merged**